### DISCUSSION TITLE
libraries conflict

### DISCUSSION THREAD

**Discussion by simontpellier** [Arduino-IRremote/Arduino-IRremote#1012] [Help] [answered]
Hello, 
I would like to report a conflict between the library and FrequencyTimer2 (perhaps others ?) using the same "cmt_isr".
Here in file "IRTimer.hpp" line 804 : _#define TIMER_INTR_NAME     cmt_isr_

regards

_C:\Users\Phi\AppData\Local\Temp\arduino_build_346938\libraries\FrequencyTimer2\FrequencyTimer2.cpp.o: In function `cmt_isr':
C:\Program-Adds\Arduino\hardware\teensy\avr\libraries\FrequencyTimer2/FrequencyTimer2.cpp:348: multiple definition of `cmt_isr'
C:\Users\Phi\AppData\Local\Temp\arduino_build_346938\sketch\teensy_IR_receive.ino.cpp.o:C:\Program-Adds\Arduino\hardware\teensy\avr\libraries\IRremote\src/IRReceive.hpp:1342: first defined here
c:/program-adds/arduino/hardware/tools/arm/bin/../lib/gcc/arm-none-eabi/5.4.1/../../../../arm-none-eabi/bin/ld.exe: Disabling relaxation: it will not work with multiple definitions
collect2.exe: error: ld returned 1 exit status
Erreur de compilation pour la carte Teensy 3.5_

**Comment by ArminJo** [comment, marked as answer]
Both libraries try to use the same Interrupt Vector called `cmt_isr`. This is a standard resource conflict. You have one Timer but 2 libraries requiring it. 😞

**Comment by simontpellier** [comment]
argh... so it's screwed !
(anyway thanks for reply)
